src/addagent/validate.c

```diff
@@ -550,9 +550,15 @@ double get_time_since_agent_disconnection(const char *id) {
         status = json_field->valuestring;
     }
 
-    json_field = cJSON_GetObjectItem(json_agent_info->child, "disconnected_time");
-    if (json_field && cJSON_IsNumber(json_field) && status && !strcmp(status, AGENT_CS_DISCONNECTED)) {
-        disconnected_time = json_field->valueint;
+    if (status) {
+        if (!strcmp(status, AGENT_CS_NEVER_CONNECTED)) {
+            return OS_INVALID;
+        } else if (!strcmp(status, AGENT_CS_DISCONNECTED)) {
```

Comment:
Code style:
Use the **else if** statemente in a new line.

Reply:
According to this convention, we are using the else clause after the "}" in the same line. 

https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C%29#style-conventions

Reply:
You are right